apps/team-awesome-web/src/components/time-entries/TimeEntries.tsx

```diff
@@ -101,6 +98,16 @@ export const TimeEntries = ({ ...props }: TimeEntriesProps) => {
     }
   };
 
+  // Inital load
+  useEffect(() => {
+    setTimeEntries(props.timeEntries);
```

Comment:
Do you need this one? Try only using the Sorted list renderer. I think that also works. So only one useEffect

Reply:
It's necessary: the `props.timeEntries` are passed by the parent where the `get-time-entries.ts` service is fired, but it has to be set to `timeEntries` because the useEffect for `sortedTimeEntries` has `timeEntries` as a dependency.